src/Crowdin.Api/Branches/BranchesApiExecutor.cs

```diff
@@ -0,0 +1,198 @@
+        {
+            _apiClient = apiClient;
+            _jsonParser = jsonParser;
+        }
+        
+        /// <summary>
+        /// Get Cloned Branch. Documentation:
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.branches.clones.branch.get">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.branches.clones.branch.get">Crowdin Enterprise API</a>
+        /// </summary>
+        [PublicAPI]
+        public async Task<Branch> GetClonedBranch(int projectId, int branchId, string cloneId)
+        {
+            var url = $"/projects/{projectId}/branches/{branchId}/clones/{cloneId}/branch";
+            CrowdinApiResult result = await _apiClient.SendGetRequest(url);
+            return _jsonParser.ParseResponseObject<Branch>(result.JsonObject);
+        }
+        
+        /// <summary>
+        /// Clone Branch. Documentation:
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.branches.clones.post">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.branches.clones.post">Crowdin Enterprise API</a>
+        /// </summary>
+        [PublicAPI]
+        public async Task<BranchCloneStatus> CloneBranch(int projectId, int branchId, CloneBranchRequest request)
+        {
+            var url = $"/projects/{projectId}/branches/{branchId}/clones";
+            CrowdinApiResult result = await _apiClient.SendPostRequest(url, request);
+            return _jsonParser.ParseResponseObject<BranchCloneStatus>(result.JsonObject);
+        }
+        
+        /// <summary>
+        /// Check Branch Clone Status. Documentation:
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.branches.clones.get">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.branches.clones.get">Crowdin Enterprise API</a>
+        /// </summary>
+        [PublicAPI]
+        public async Task<BranchCloneStatus> CheckBranchCloneStatus(int projectId, int branchId, string cloneId)
+        {
+            var url = $"/projects/{projectId}/branches/{branchId}/clones/{cloneId}";
+            CrowdinApiResult result = await _apiClient.SendGetRequest(url);
+            return _jsonParser.ParseResponseObject<BranchCloneStatus>(result.JsonObject);
+        }
+        
+        /// <summary>
+        /// List Branches. Documentation:
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.branches.getMany">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.branches.getMany">Crowdin Enterprise API</a>
+        /// </summary>
+        [PublicAPI]
+        public async Task<ResponseList<Branch>> ListBranches(
```

Comment:
Some of these branch methods are already included in the source files namespace, we need to keep the existing without duplicating